packages/SwingSet/analysis-tools/db-dump.js

```diff
@@ -0,0 +1,52 @@
+import '@agoric/install-ses';
+import process from 'process';
+import { openLMDBSwingStore } from '@agoric/swing-store-lmdb';
```

Comment:
This appears to have been cribbed from one of the db tools as they existed before the recent swing store refactoring.